src/presentation/http/router/note.test.ts

```diff
@@ -100,16 +120,18 @@ describe('Note API', () => {
         url: `/note/${note.publicId}`,
       });
 
-      expect(response?.statusCode).toBe(200);
+      expect(response?.statusCode).toBe(expectedStatusCode);
 
-      expect(response?.json()).toMatchObject({
-        'note': {
-          'id': note.publicId,
-        },
-        'accessRights': {
-          'canEdit': true,
-        },
-      });
+      if (expectedStatusCode === 200) {
+        expect(response?.json()).toMatchObject({
+          'note': {
+            'id': note.publicId,
+          },
+          'accessRights': {
+            'canEdit': canEdit,
+          },
+        });
+      }
```

Comment:
let add `else` with checking that unsuccessful request has no accidentally leaked data in response (response should be empty)